### ISSUE TITLE
Option to include or exclude "today" from Last N filters

### ISSUE THREAD

**Issue by gauxgaux** [RevealBi/Reveal.Sdk#42]
**Is your feature request related to a problem? Please describe.**
We have had multiple customer complaints that when people select a Last N filter it includes more days than they expect. eg. the Last 7 days includes 8 because it includes today. 

**Describe the solution you'd like**
If there is a business use for having the filters as is, perhaps there could be a way to give an option to include or exclude today from the Last N calculations. 

**Describe alternatives you've considered**
The active workaround right now is to use the between 2 dates filter. The drawback to this is you have to change the every time you open the report, versus the filters being always set for a Last N.

**Additional context**
Nope!

**Comment by VANSHIKAAGARWAL571** [comment]
Yes this problem is coming with the reports when user selects last 7 days it shows 8 days data including current date. So need this issue to get resolved as soon as possible.

**Comment by MLopezIG** [comment]
Hi @gauxgaux and @VANSHIKAAGARWAL571 

From the description of the issue, looks like the main problem here is that "Last N days" ranges return N+1 days, which we agree is confusing. 

It's true that by excluding 'today' from the current range will result in N days being returned, which seems more appropriate, but an alternative is include today, but still return only N days in total. 

So the question here is if either solution would work for you, or if you really need, for whatever reason, to exclude Today from the returned range.

**Comment by brianlagunas** [comment]
@gauxgaux @VANSHIKAAGARWAL571 

I'd like to follow up and see if you had a response to the question @MLopezIG asked? We are trying to work on this feature, but need to clarify your requirements.

**Comment by brianlagunas** [comment]
This has been implemented in our latest 1.7.2 release. Last 7 days will actually return the last 7 days including today.
